app/views/dashboard.pug

```diff
@@ -12,69 +12,72 @@ html(lang="en")
         script(type='text/javascript' src='/dashboard.js')
 
     // Main Body
-    body(style='background-image:url("/assets/background.png");')
+    body
+        div.container-fluid#content
 
-        // Main Header
-        div(id='header')
-            div(id='mainTitle' style='text-align:center;width:80%;')
-                h1 Welcome to Hive!
+            // Side Button Bar

```

Comment:
Something is currently off visually (with the css) with the nav bar.  I say we wait until Lincoln's pr is merged then try and come back and fix it.